src/Auth0.OidcClient.Core/Tokens/IdTokenRequirements.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+
+namespace Auth0.OidcClient.Tokens
+{
+    /// <summary>
+    /// Identity token validation requirements for use with <see cref="IdTokenValidator"/>.
+    /// </summary>
+    internal class IdTokenRequirements
+    {
+        /// <summary>
+        /// Required issuer (iss) the token must be from.
+        /// </summary>
+        public string Issuer;
+
+        /// <summary>
+        /// Required audience (aud) the token must be for.
+        /// </summary>
+        public string Audience;
+
+        /// <summary>
+        /// Optional one-time nonce the token must be issued in response to.
+        /// </summary>
+        public string Nonce;
+
+        /// <summary>
+        /// Optional maximum time since the user authenticated, in seconds.
+        /// </summary>
+        public TimeSpan? MaxAge = null;
```

Comment:
Just making sure, option to send Max age is done through OidcClient right?

Reply:
It's just a value that the developer can specify and we check client side that `auth_time` isn't too old based on what they specified in `MaxAge`. Should we be sending something to the server?

Reply:
@damieng George question was whether the developer will be able to send this max age value on the authentication request or not; and if that goes through the OidcClient